src/components/SidebarLayout/Header/components/SafeListHeader/index.tsx

```diff
@@ -1,3 +1,5 @@
+// TODO: SHOULD BE REMOVED (NOT USED ANYMORE)
```

Comment:
Is the whole component not used because design changes?

When is it planned to be removed?

Reply:
removed